File: model/src/main/java/no/nav/pensjon/vtp/testmodell/organisasjon/OrganisasjonRepository.java
package no.nav.pensjon.vtp.testmodell.organisasjon;

import java.util.Optional;

public interface OrganisasjonRepository {
    Optional<OrganisasjonModell> findById(String orgnr);

    void saveAll(Iterable<OrganisasjonModell> modeller);
}

File: model/src/main/java/no/nav/pensjon/vtp/testmodell/organisasjon/OrganisasjonRepository.kt
package no.nav.pensjon.vtp.testmodell.organisasjon

import org.springframework.data.repository.Repository

interface OrganisasjonRepository : Repository<OrganisasjonModell, String> {
    fun findById(orgnr: String): OrganisasjonModell?
    fun saveAll(modeller: Iterable<OrganisasjonModell>)
}
